client/validators/validateUseremail.ts

```diff
@@ -2,7 +2,7 @@ import { z } from 'zod';
 
 const disposableEmailDomains = [
   'mailinator.com',
-  'dispostable.com',
+  'dispostable.com', // is this a typo?
```

Comment:
not a typo, but I think this service is deprecated. A quick search shows these are popular: 
https://maildrop.cc/  
https://maildrop.c 
https://incognitomail.co

Reply:
https://www.npmjs.com/package/disposable-email-detector